packages/react-components/react-timepicker-compat-preview/src/components/TimePicker/TimePicker.types.ts

```diff
@@ -116,6 +114,18 @@ export type TimePickerProps = Omit<
     | 'freeform'
   > &
   TimeFormatOptions & {
+    /**
+     * TimePicker has default children. It can be customized as either:
+     * 1. A single element or an array of elements
+     * 2. A render function that will receive properties and must return valid element(s) or null
+     * 3. null or undefined
+     */
+    children?:
```

Comment:
is it a requirement to have custom children in the TimePicker?

Reply:
Nope, but I think it's nice to have. Do you think we should hold off until someone request a feature?

Reply:
Yes, it wasn't a feature in v8, so let's wait until it's requested